### ISSUE TITLE
use pylogeny constructs for data handling

### ISSUE THREAD

**Issue by LinguList** [pylogeny/concordance#1]
we have pylogeny.tree, so we don't need dendropy.tree, we also have pylogeny.parsimony, so don't need our own implementation of parsimony here

**Comment by PhyloStar** [comment]
Yes. One question is w.r.t the weighted parsimony on a quartet. What is the best strategy to compute weighted parsimony on a quartet? Should we just pass a newick tree like `((A, B), (C, D));`?

@WesScivetti Can you look at modifying the parsimony function call `fitch_pars` on lines 87, 89, and 91 to `def parsimony(tree, pattern, characters=None, matrix=None):`

**Comment by LinguList** [comment]
Parsimony on quartets can be done in a new extra function, I'd like to follow @xrotwang's note that we need clear functions with clear ideas of what to send to them and what we get back, and we should work with simple Python objects here. But for a quartet, it may be faster to not use our parsimony code. Can you specify how you would expect the function to work here? Note also that we would want to use different models for this.

**Comment by LinguList** [comment]
I mean also: for the current funciton, could you give me an input output example?

**Comment by LinguList** [comment]
So, @WesScivetti, let us start with an example of only this function, and input and output, so we understand more properly, what this function does here.

**Comment by LinguList** [comment]
In general, the current code does not clearly divide functions and reading and writing files, so it makes it hard to test it or to dig deeper here, which is also why I made the quartet function from scratch, as I could not test the one in the current code with dendropy, which is not easy in tree handling...
